packages/browser-sdk/src/ProviderManager.ts

```diff
@@ -118,7 +118,7 @@ export class ProviderManager implements EventEmitter {
 
     if (requestedProvider === "injected") {
       targetProviderType = "injected";
-    } else if (["google", "apple", "jwt", "phantom"].includes(requestedProvider)) {
+    } else if (["google", "apple", "phantom", "tiktok", "x"].includes(requestedProvider)) {
```

Comment:
What do you think about using an enum for the `EmbeddedProviderAuthType` going forward? This check is easy to miss, and is in two places. It could be simplified and forgotten about, e.g.

```ts
function isEmbeddedProviderAuthType(provider: string): provider is EmbeddedProviderAuthType {
  return Object.values(EmbeddedProviderAuthType).includes(requestedProvider)
}
```